### DISCUSSION TITLE
Seeking to customize formatting of info in yaml for title page of docx - hoping someone can point me in right direction

### DISCUSSION THREAD

**Discussion by donboyd5** [quarto-dev/quarto-cli#692] [Q&A]
Hi. I would like the cover (i.e., first) page of a docx report to be a bit customized. In particular, I have 3 things I would like to do, the first of which I have a workaround for:

First: 
- I see that I can use a reference doc to enforce a first-page header with logo in the first page of the output docx. That's really great.
- However, the logo is taller than the typical top margin. If the report title (in _quarto.yml) starts right after the top margin, it is too high up on the page, almost overlapping the logo. 
- I would like to push the title down an inch or so from the logo. I tried putting carriage returns (\n\n or `<br><br>`) in the title text in the yaml but that doesn't work.
- I tried putting carriage returns in the body of the reference doc, but I am not surprised that doesn't work, because its text content is overwritten or discarded when the docx is rendered.
- My workaround - which works well - was to put carriage returns in the first-page header to create space in the header below the logo. That works fine.
- However - and obviously this is a minor wish - it would be nice to be able to control header space programmatically rather than in the header. Not a big deal but nice to have. Someday.

Second:

- I would like to be able to place the author and date fields lower down in the document, so that they are just above the footer. I prefer not to find a hard-coded author-date reference doc solution to this, possibly adjusting the first-page footer, because I would like the date to be determined based on when the document is rendered.
- Can anyone suggest an approach that would let me push the author and date fields down in a rendered docx?

Third:

- The title is too long for a single line, and wraps. I would like to control where it wraps, because it has a city name in it ("San Francisco") and I don't want the break to occur between "San" and "Francisco", where it naturally will break -- I want to keep "San Francisco" together on the same line of the title.
- Can anyone point me to an approach that would do this?

Many thanks.

**Comment by rbcavanaugh** [comment]
I'm working with some similar constraints from quarto --> word and have a few suggestions: 

For #1: You can modify the spacing of the 'title' style in your word document. For example, if you add '24pt' above to the title style in the reference document, it will push the title down. 

For #2: Same strategy works - add spacing to the author/date fields. This probably isn't a perfect solution to push these fields to the bottom of the page, since multi-line titles might push these fields too far. 

For #3: You can break the title onto multiple lines as follows:

```
title: |
   This is the first line of my title: \
   its a very good title
```

**Comment by jjallaire** [comment]
@dragonstyle I think we'll want to do custom title blocks for docx (seems possible given https://github.com/jgm/pandoc-templates/blob/master/default.opendocument)
